scripts/exglobal_atmos_vminmon.sh

```diff
@@ -19,39 +19,11 @@ source "$HOMEgfs/ush/preamble.sh"
 #
 ################################################################################
 
-
-########################################
-#  Set environment
-########################################
-export NET=${NET:-gfs}
-export RUN=${RUN:-gdas}
-export envir=${envir:-prod}
-
 ########################################
-#  Directories
-########################################
-export DATA=${DATA:-$(pwd)}
-
-
+#  Command line arguments
 ########################################
-#  Filenames
-########################################
-gsistat=${gsistat:-${COM_ATMOS_ANALYSIS}/gdas.t${cyc}z.gsistat}
-export mm_gnormfile=${gnormfile:-${M_FIXgdas}/gdas_minmon_gnorm.txt}
-export mm_costfile=${costfile:-${M_FIXgdas}/gdas_minmon_cost.txt}
-
-########################################
-#  Other variables
-########################################
-export MINMON_SUFFIX=${MINMON_SUFFIX:-GDAS}
-export PDATE=${PDY}${cyc}
-export NCP=${NCP:-/bin/cp}
-export pgm=exgdas_vrfminmon.sh
-
-if [[ ! -d ${DATA} ]]; then
-   mkdir $DATA
-fi
-cd $DATA
+export PDY=${1:-${PDY:?}}
+export cyc=${2:-${cyc:?}}
```

Comment:
This will be set in the j-job already.  
We don't do this in any exscript, so why add here?